### DISCUSSION TITLE
Suggestion: Showering

### DISCUSSION THREAD

**Discussion by Kegdo-Jr** [cataclysmbn/Cataclysm-BN#2877] [General]
### Is your feature request related to a problem? Please describe.

The feature most players actually need in both cata and real life, ong
There is no early-game method to hide a player's scent, only bionics(!!!) can do that. Why not just splash water with soap on your body for the nearly-same effect?

### Describe the solution you'd like

New craftable item - showering kit, crafted from a brush, sponge and a towel, uses soap as ammo (5 per use, 10 charges total).
On activation, the item should look for 20 units of water in the player's nearby vicinity, clean or not matters not.
The item should only be usable while nude (to prevent casually coating yourself with water in sub-0 degree temperatures in winter and NOT dying instantly) and call a 30-minute activity that, upon completion:
Slightly buffs the hidden health variable
Moderately buffs mood
Reduces scent level and scent produced by 90%
Leaves a puddle of wa'er on the ground for le environmental storytelling
Slightly cools the player?

The duration would be from 7 to 12 hours, needs discussion.

### Additional context

You can also just remove scent. Might be an even better solution.

**Comment by scarf005** [comment]
only being usable while nude could mean lots of additional keypress, instead we could
1. calculate time to take off all clothing, and implicitly add `(time took to take off all clothing) * 2` to time.
pros: easy to implement.
cons: it gets awkward when you get interrupted as you've been showering while clothed.
2. save all clothing info, automatically take off all clothing, then do the reverse.
pros: most accurate.
cons: harder to implement. thus, after being interrupted you have to manually wear the clothes.

**Reply by Kegdo-Jr** [reply]
I think 2 isn't that hard to do, despite taking more time to code than 1. It's better to have a robust system instead of quickly slapping on something that isn't even versatile.

The activity should also have a warning about going commando, since this is a long period of time that leaves the player vulnerable.

**Reply by Zireael07** [reply]
I don't think this will happen without some significant QoL improvements @scarf005 mentions.

Been suggested many times on the original DDA and rejected for adding tedium, and I think BN is even more tedium-averse
